resource-managers/mesos/pom.xml

```diff
@@ -29,7 +29,7 @@
   <name>Spark Project Mesos</name>
   <properties>
     <sbt.project.name>mesos</sbt.project.name>
-    <mesos.version>1.0.0</mesos.version>
+    <mesos.version>1.3.0-rc1</mesos.version>
```

Comment:
1.3.0 is out I think right? Should we depend on a stable release? Does this change anything?